stencils.html

```diff
@@ -383,6 +383,11 @@ <h2>Talks</h2>
                                 <p><a href="presentations/2017/CArD-2017.pdf">
                                     Performance Portable Stencil Code Generation in LIFT</a></p>
                             </li>
+                            <li>

```

Comment:
Could you move this above the 2017 talk? So that we have the most recent talks first.